src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,168 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    static class Node<T> {
+        private T element;
+        private Node<T> next;
+        private Node<T> prev;
+
+        private Node(T element) {
+            this.element = element;
+        }
+    }
```

Comment:
```suggestion
    private static class Node<T> {
        private T element;
        private Node<T> next;
        private Node<T> prev;

        public Node(T element) {
            this.element = element;
        }
    }
```
dont forget about access modifiers
and move after public methods